accounts-db/src/append_vec.rs

```diff
@@ -724,6 +909,13 @@ impl AppendVec {
                     result.push(next.stored_size_aligned);
                 }
             }
+            AppendVecFileBacking::FileOnly(_file) => {
+                for &offset in sorted_offsets {
+                    self.get_stored_account_meta_callback(offset, |stored_meta| {
+                        result.push(stored_meta.stored_size());
```

Comment:
yes, this could be done more efficiently. we still have to read the data len and then move and align the next offset. We don't have to read the `data` itself into a buffer, so we can do what `scan_index` does. I think this would be a great fun project for someone after this.